docs/docusaurus/docs/core/expectation_suites/manage_expectation_suites.md

```diff
@@ -42,13 +42,14 @@ You can add an Expectation Suite to your Data Context at the same time as you cr
 
 :::
 
-<details><summary>Full example code</summary>
```

Comment:
I tried a few things. This first change is structured differently than the rest, and both seem to pass.